cmd/export_ledgers.go

```diff
@@ -62,7 +67,7 @@ var ledgersCmd = &cobra.Command{
 		// TODO: check the permissions of the file to ensure that it can be written to
 		outFile, err := os.OpenFile(absolutePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
 		if err != nil {
-			logger.Fatal("could not create output file: ", err)
+			logger.Fatal("could not open output file: ", err)
```

Comment:
This error message should communicate which command is having the problem. But there's a chance that it isn't an open issue, so this message could be deceptive. Suggestion: error in output file for <command>: